test/src/api/assertions/testEnabled.js

```diff
@@ -3,15 +3,15 @@ const Globals = require('../../../lib/globals.js');
 
 xdescribe('assert.enabled', function () {
```

Comment:
you need to re-enable this, `xdescribes` means that it would be skipped.

Reply:
Done ✅